cmd/node/config.go

```diff
@@ -50,3 +52,16 @@ func (cfg *NodeConfig) allFieldsPresent() error {
 	}
 	return nil
 }
+
+func (config *NodeConfig) generateNodeID() (*[]byte, error) {
```

Comment:
@stepanvalyavskiy go slices has a reference to underlying array, so it's redundant to use reference to slice, you can just return `[]byte`, see https://stackoverflow.com/a/39993797/1723695